python/pyarrow/_parquet.pyx

```diff
@@ -1334,6 +1339,15 @@ cdef class ParquetWriter:
             for column in self.use_dictionary:
                 props.enable_dictionary(column)
 
+    cdef void _set_byte_stream_split_props(
+            self, WriterProperties.Builder* props):
+        if isinstance(self.use_byte_stream_split, bool):
+            if self.use_byte_stream_split:
+                props.encoding(ParquetEncoding_BYTE_STREAM_SPLIT)
+        elif self.use_byte_stream_split is not None:
+            for column in self.use_dictionary:
```

Comment:
Looks like this should be `self.use_byte_stream_split`?

Reply:
Done, thanks.